cylc/flow/loggingutil.py

```diff
@@ -238,15 +239,22 @@ def do_rollover(self) -> None:
             os.dup2(self.stream.fileno(), sys.stderr.fileno())
         # Emit header records (should only do this for subsequent log files)
         for header_record in self.header_records:
+            now = time()
```

Comment:
IMO it's entirely reasonable not to test assigning the output of a standard library function, and marking this as not needing testing gives us a better idea of what is and isn't tested.

```suggestion
            now = time()   # pragma: no cover
```

Reply:
I'm confused, codecode says this is covered:

> [codecov/patch](https://github.com/cylc/cylc-flow/pull/6546/checks?check_run_id=35367886718) — 100.00% of diff hit (target 97.00%)

Reply:
![image](https://github.com/user-attachments/assets/8d5095fa-3e86-4ac7-b996-680ec2549be3)

Reply:
![Screenshot from 2025-01-15 16-36-48](https://github.com/user-attachments/assets/8cce7510-32c9-4ee9-b2d4-37aa7906ba78)